### DISCUSSION TITLE
Add new exchanges??

### DISCUSSION THREAD

**Discussion by Zourvan** [oliver-zehentleitner/unicorn-binance-websocket-api#278] [General]
Hi
Is any decision to expand this library to other exchanges?
may list full documentation to add new exchange?

**Comment by oliver-zehentleitner** [comment]
I have been looking at the idea for Kucoin myself. I think a fork and an adaptation would make the most sense. One module per Exchange instead of all in one.

The problem with this is rather the big effort and the funding....
